src/v0/util/index.js

```diff
@@ -1419,7 +1419,7 @@ function generateUUID() {
   }
   return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, (c) => {
     // eslint-disable-next-line no-bitwise
-    const r = (d + Math.random() * 16) % 16 | 0;
+    const r = (d + crypto.randomBytes(1)[0]) % 16 | 0; 
```

Comment:
https://www.geeksforgeeks.org/node-js-crypto-randomuuid-function/

already there is direct function available https://nodejs.org/api/crypto.html#cryptorandomuuidoptions